src/components/atoms/Loader/index.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import './style.css';
```

Comment:
I decided to redeclare a `Loader` component here as the `spinner` one cannot be used as is.  It needs a div wrapper with fixed height for instance, to display correctly. 

I believe we should always avoid such components that need external div wrapper in order to display correctly. At least this div wrapper should be included in the component and parametrized with props or higher order compoenents